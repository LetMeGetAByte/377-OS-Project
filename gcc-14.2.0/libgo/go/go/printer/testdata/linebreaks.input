version https://git-lfs.github.com/spec/v1
oid sha256:2b279bf52c00fb54aeefc24a17eb886c5b3c4d90492b6a351735b00302cf6813
size 5189
