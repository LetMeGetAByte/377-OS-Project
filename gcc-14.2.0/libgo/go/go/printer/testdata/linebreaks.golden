version https://git-lfs.github.com/spec/v1
oid sha256:ec85b3cecd5c6a830fd10750a51e2186296907da0b6274b2cc9282722e45483b
size 5217
